services/eventservice/src/tests.rs

```diff
@@ -0,0 +1,268 @@
+use crate::types::{Category, Event};
+
+use super::*;
+
+fn test_categories() -> (Category, Category, Category, Category) {
+    let cat1 = "first".to_string();
+    let cat2 = "second".to_string();
+    let cat3 = "third".to_string();
+    let cat4 = "fourth".to_string();
+    (cat1, cat2, cat3, cat4)
+}
+
+fn test_events(cat1 : Option<Category>, cat2 : Option<Category>,cat3 : Option<Category>) -> (Event,Event,Event) {
+    let (tcat1, tcat2, tcat3, _) = test_categories();
+    let event1 = types::Event {
+        id: 1,
+        title: "Event1".to_string(),
+        cover: "N/A".to_string(),
+        description: "Empty".to_string(),
+        organizer: 3,
+        date_created: chrono::Utc::now().timestamp(),
+        date_planning: (chrono::Utc::now() + chrono::Duration::days(3)).timestamp(),
+        category: cat1.unwrap_or(tcat1)
+    };
+    let event2 = types::Event {
+        id: 2,
+        title: "Event2".to_string(),
+        cover: "N/A".to_string(),
+        description: "Empty".to_string(),
+        organizer: 1,
+        date_created: chrono::Utc::now().timestamp() + 5,
+        date_planning: (chrono::Utc::now() + chrono::Duration::days(2)).timestamp(),
+        category: cat2.unwrap_or(tcat2)
+    };
+    let event3 = types::Event {
+        id: 3,
+        title: "Event3".to_string(),
+        cover: "N/A".to_string(),
+        description: "Empty".to_string(),
+        organizer: 2,
+        date_created: chrono::Utc::now().timestamp() + 10,
+        date_planning: (chrono::Utc::now() + chrono::Duration::days(1)).timestamp(),
+        category: cat3.unwrap_or(tcat3)
+    };
+    (event1, event2, event3)
+}
+
+#[test]
+fn merge_test() {
+    assert_eq!(
+        data_manager::merge(b"", Some(b"ab"), b"cd"),
+        Some((b"abcd").to_vec())
+    );
+}
+
+
+#[test]
+fn create_event_test() {
+    let manager = data_manager::EventManager::new(&"./test/createeventtestdb".to_string()).expect("Failed to create db");
```

Comment:
This creates a real db that didn't cleaned up after the test. 

Probably, we need a mock object for this, or delete the db after test. `reset_all` method just cleans data, without removing them from the filesystem.

Reply:
Created method to clean folder